client/src/components/shared/restaurant/restaurant-item.js

```diff
@@ -1,9 +1,9 @@
 import React, { Component } from 'react';
 import { Tab, Tabs, Container } from 'react-bootstrap';
 import PropTypes from 'prop-types';
-import Api from '../services/api';
+import Api from '../../../services/api';
 import About from './restaurant-about';
-import '../style/restaurant-item.css';
+import '../../../style/restaurant-item.css';
 
 class Restaurant extends Component {
   constructor(props) {
```

Comment:
I don't understand what happens and why
![image](https://user-images.githubusercontent.com/6152066/77223475-a4dc6e80-6b65-11ea-8c23-550b5e6413f6.png)

Reply:
> As you send a lot of constants in separate files (I think it is bad idea if we use this constants only in one places each) you shouldn't place those files in component folder. You can create `/client/src/shared/constants` folder, or something similar. It makes your code structure simplier
> 
> I think this code brings more difficulties in our project. We have very few entities (`Restautant and Events`) but we will get very huge and difficulty structure.
> 
> ```
> client/src
> ├── app.js
> ├── app.test.js
> ├── components
> │   ├── info
> │   │   ├── events.js
> │   │   ├── navigation.js
> │   │   └── rotes.js
> │   ├── openlayers-map.js
> │   └── shared
> │       ├── button
> │       │   ├── date-picker.js
> │       │   ├── reset.js
> │       │   └── search-menu-button.js
> │       ├── dropdown
> │       │   ├── context.js
> │       │   ├── date-menu.js
> │       │   └── view.js
> │       ├── event
> │       │   ├── card-results.js
> │       │   └── card-view.js
> │       ├── navigation
> │       │   ├── navigation-bar.js
> │       │   ├── navigation-item.js
> │       │   └── navigation-route.js
> │       ├── restaurant
> │       │   ├── restaurant-about.js
> │       │   └── restaurant-item.js
> │       └── search
> │           ├── filter.js
> │           ├── header.js
> │           ├── input.js
> │           ├── search-menu.js
> │           └── title.js
> ├── index.js
> ├── pages
> │   ├── about.js
> │   ├── contact.js
> │   ├── events.js
> │   ├── home.js
> │   ├── login.js
> │   ├── map.js
> │   ├── no-match.js
> │   ├── registration.js
> │   └── restaurant-list.js
> ├── services
> │   └── api.js
> ├── service-worker.js
> ├── setup-tests.js
> └── style
>     ├── app.css
>     ├── dropdown.css
>     ├── events-page.css
>     ├── m-button.css
>     ├── openlayers-styles.css
>     ├── restaurant-about.css
>     └── restaurant-item.css
> ```
> 
> I think this code won't make my developer experience simpler and we should re-think about this task

Check Step 1 **[here](https://reactjs.org/docs/thinking-in-react.html)** and **[this article](https://medium.com/@level_out/everything-is-a-component-cf9f469ad981)**, please.